src/Symfony/Component/Form/Resources/translations/validators.bg.xlf

```diff
@@ -14,6 +14,126 @@
                 <target>Невалиден CSRF токен. Моля, опитайте да изпратите формата отново.</target>
             </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Стойността не е валиден HTML5 цвят.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Моля въведете валидна дата на раждане.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>Избраните стойности не са валидни.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>Колекцията не е валидна.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Моля изберете валиден цвят.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Моля изберете валидна държава.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Моля изберете валидна валута.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Моля изберете валиден интервал от дати.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Моля въведете валидни дата и час.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Моля въведете валидна дата.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Моля изберете валиден файл.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Скритото поле е невалидно.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Моля попълнете цяло число.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Моля изберете валиден език.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Моля изберете валидна локация.</target>
```

Comment:
Can you double check if this is correct. 

A locale is "sv", "en", "fr", etc.

Reply:
I think this is okay. There is no exact translation for locale in bulgarian.

Reply:
Okey. Cool cool. 

Thank you

Reply:
But,, when I use google translate, it says "location", is google translate wrong? Or will it be better to say "език"?

Reply:
Език means language, but probably here will work also :)

Reply:
Yeah. Because the English word "Location" is wrong. 

If "локация" refers to "sv", "en", "fr", etc, then Im happy with this.

Reply:
Локация refers to Location

Reply:
Okey, I think we need to change it then. Because a location is a place, like Stockholm, London or "under the table". 

If there is no word for "locale" in Bulgarian then we could use the Bulgarian word for "language" instead.